### PR TITLE
chore(pylint) : Ignore module doc string from pylint

### LINKED ISSUE #849: Make PyLint ignore Module Docstring
# Description 
The PyLint report generates an error ```C0114 (missing-module-docstring)``` for most of the Python files.

We don't need this sanity check for our Repo.

So we can make sure that PyLint ignores it.

**Expected behavior**
PyLint should ignore Missing DocString.

**Additional context**
The solution is rather simple and requires some small edits it in this [config file](https://github.com/HarshCasper/Rotten-Scripts/blob/master/.github/linters/.python-lint.yml)

For reference here is a [thread](https://stackoverflow.com/questions/7877522/how-do-i-disable-missing-docstring-warnings-at-a-file-level-in-pylint).

### PR DESCRIPTION
**Pull request by priyanshu20**
# Description

Ignored alert from pylint about module docstring

Fixes #849 

## Type of change

- [x] Documentation Update

# Checklist:

- [ ] My code follows the style guidelines(Clean Code) of this project
- [ ] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have created a helpful and easy to understand `README.md`
- [ ] I have made corresponding changes to the documentation
- [ ] My changes generate no new warnings
- [ ] I have added tests/screenshots(if any) that prove my fix is effective or that my feature works

### CODE REVIEW

**Comment by vybhav72954** [comment]
@priyanshu20 While you are at this, can you add a few more codes as well. I will post the codes here.

**Comment by priyanshu20** [comment]
> @priyanshu20 While you are at this, can you add a few more codes as well. I will post the codes here.

sure

**Comment by vybhav72954** [comment]
Add these codes

```
C0103
R0914
W0702
C0411
```

File: .github/linters/.python-lint.yml [hunk 1]
```diff
@@ -38,7 +38,7 @@ load-plugins=
 # --enable=similarities". If you want to run only the classes checker, but have
 # no Warning level messages displayed, use"--disable=all --enable=classes
 # --disable=W"
-disable=F0401,W0141,W0611,W0110,W0703,W0142,C0114
+disable=F0401,W0141,W0611,W0110,W0703,W0142,C0114,C0103,R0914,W0702,C0411
```
Comment (vybhav72954) [inline .github/linters/.python-lint.yml:41]: Final Tags Requested:
```
E1101
C0305
R0201
R0903
W0105
W0621
W0702
W0612
W0613
```

**Review by vybhav72954 — CHANGES_REQUESTED** [review]

**Comment by vybhav72954** [comment]
@HarshCasper This is good to go!

**Review by vybhav72954 — APPROVED** [review]

**Pull request merged**